tests/auth-flow-.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { expect, test } from '@playwright/test'
+import { StatusCodes } from 'http-status-codes'
+import { LoginDto } from './dto/login-dto'
+
+const serviceURL = 'https://backend.tallinn-learning.ee/'
+const loginPath = 'login/student'
+
+test('Incorrect login and password', async ({ request }) => {
+  const loginDto = LoginDto.createLoginWithICorrectCredentials()
+  const response = await request.post(`${serviceURL}${loginPath}`, {
+    data: loginDto,
+  })
+
+  console.log('response status:', response.status())
```

Comment:
Just a minor thing, but I would in general remove console logs from code before making PR, otherwise in a big project they build up over time and clutter the console.

Reply:
file name tests/auth-flow-.spec.ts is also should be fixed

Reply:
Fixed